muzlag/pkg/ffmpeg/channel.go

```diff
@@ -0,0 +1,18 @@
+package ffmpeg
+
+import "strconv"
+
+type Channel int
+
+const (
+	MonoChannel   Channel = 1
+	StereoChannel Channel = 2
```

Comment:
nit(no-op): you can write something like that

```suggestion
	MonoChannel   Channel = iota + 1
	StereoChannel
```